interzeptor.go

```diff
@@ -0,0 +1,140 @@
+// Package interzeptors constructs a gRPC interceptor to enforce gNSI services.
+package interzeptors
+
+import (
+	"context"
+
+	apb "github.com/openconfig/gnsi/acctz"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/interzeptor/acctz"
+	"google.golang.org/grpc/interzeptor/authz"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	dlog = logrus.WithField("denvelope", "interzeptors")
+
+	zeptor *Interzeptor
+)
+
+type interzeptor interface {
```

Comment:
Suggest to add a comment for the `interface` here -- since I expect you're going to expand this as you add support for other *zs?